### PR TITLE
Roistat Trace Added

### PR DESCRIPTION
**Pull request by dhavalveera**
### Technology Request

**Examples:**
https://foma.ru/
https://qwintry.com/en/in
https://weblium.com/
https://progressme.ru/
https://www.pribortorg.by/en
https://xtern.ru/

### CODE REVIEW

File: src/technologies/r.json [hunk 1]
```diff
@@ -1907,6 +1907,20 @@
...
+    "cats": [
+      10,
+      74
+    ],
+    "description": "Roistat is a cross-cutting analytics system, collects data from CRM-system, advertising platforms.",
+    "icon": "roistat.png",
+    "pricing": [
+      "low",
+      "poa"
+    ],
```
Comment (nurbek91) [inline src/technologies/r.json:1920]:
```
 "pricing": [
      "mid",
      "recurring",
      "poa"
    ],
```

File: src/technologies/r.json [hunk 2]
```diff
@@ -1907,6 +1907,20 @@
...
+    "scripts": "cloud\\.roistat\\.com",
```
Comment (nurbek91) [inline src/technologies/r.json:1921]: intsead
```
 "js": {
      "roistatHost": "",
      "roistatProjectId": ""
    },
```

**Review by nurbek91 — CHANGES_REQUESTED** [review]
Hey mate, thanks. need small updates

**Review by nurbek91 — APPROVED** [review]

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**